package com.todo.todoapplication.domain.user.exception

const val USER_NOT_FOUND_MESSAGE = "해당 유저가 존재하지 않습니다."
